x-pack/plugins/actions/server/create_execute_function.ts

```diff
@@ -151,6 +155,18 @@ export function createBulkExecutionEnqueuerFunction({
       );
     }
 
+    if (
+      await hasReachedTheQueuedActionsLimit(
+        taskManager,
+        configurationUtilities,
+        actionsToExecute.length
+      )
+    ) {
+      throw new Error(
```

Comment:
Can we create a custom error that can be checked by the caller? That would allow the calling fn to distinguish between this specific error (and possibly initiate a retry) and other errors like the ESO error

Reply:
Resolved in this commit https://github.com/elastic/kibana/pull/164632/commits/539da5a45f47c0c95a3cbfdd7fdbcbcd70ad2330